core/src/test/java/uk/gov/justice/services/core/json/FileBasedJsonSchemaValidatorTest.java

```diff
@@ -34,6 +36,10 @@
     @Mock
     private PayloadExtractor payloadExtractor;
 
+    @Mock
+    private DefaultJsonValidationLoggerHelper defaultJsonValidationLoggerHelper;
```

Comment:
This can be injected as a JsonValidationLoggerHelper

Reply:
JsonValidationLoggerHelper does not have knowledge of ValidationExceptions so cannot work here.